From 324dcac505e1c2c3f313392769713cff2aec2d59 Mon Sep 17 00:00:00 2001
From: Stanislav Ochotnicky <sochotnicky@redhat.com>
Date: Mon, 17 May 2010 13:59:46 +0200
Subject: [PATCH 06/10] Remove mvn-clirr-plugin

for now this is included in current mvn in Fedora. Remove once
clirr-plugin is separate
---
 pom.xml |   15 ---------------
 1 files changed, 0 insertions(+), 15 deletions(-)

diff --git a/pom.xml b/pom.xml
index c273581..6e32bb7 100644
--- a/pom.xml
+++ b/pom.xml
@@ -350,21 +350,6 @@ target platform and specify -Dntdll_target=msbuild on the mvn command line.
           </execution>
         </executions>
       </plugin>
-      <!--
-        clirr:check will fail with NullPointerException
-        due to missing javax.jms.MessageListener,
-        however it will trigger download of supporting components
-        which can allow "ant clirr" to succeed.
-        Could possibly run on a JavaEE platform.
-       -->
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>clirr-maven-plugin</artifactId>
-        <version>2.2.2</version>
-        <configuration>
-          <comparisonVersion>1.2.15</comparisonVersion>
-        </configuration>
-      </plugin>
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>rat-maven-plugin</artifactId>
-- 
1.6.6.1

